Feature/BoletoMultaeJuros-Choapi (#1919)
mercadopago/devsite-docs: guides/checkout-api-v2/boleto-bancario.en.md

Mercadopago.configurations.setAccessToken(config.access_token);

var response = await Mercadopago.payment_methods.listAll();
var payment_methods = response.body;
```
```java
MercadoPagoConfig.setAccessToken("ENV_ACCESS_TOKEN");

PaymentMethodClient client = new PaymentMethodClient();
client.list();

```
```ruby
require 'mercadopago'
sdk = Mercadopago::SDK.new('ENV_ACCESS_TOKEN')

payment_methods_response = sdk.payment_methods.get()
payment_methods = payment_methods_response[:response]

```
```csharp
using MercadoPago.Client.PaymentMethod;
using MercadoPago.Config;
using MercadoPago.Resource;
using MercadoPago.Resource.PaymentMethod;

MercadoPagoConfig.AccessToken = "ENV_ACCESS_TOKEN";

var client = new PaymentMethodClient();
ResourcesList<PaymentMethod> paymentMethods = await client.ListAsync();

```
```python
import market
sdk = Mercadopago.SDK("ACCESS_TOKEN")

payment_methods_response = sdk.payment_methods().list_all()
payment_methods = payment_methods_response["response"]
```
```curl
curl -X GET \
-H 'accept: application/json' \
-H 'content-type: application/json' \
-H 'Authorization: Bearer ENV_ACCESS_TOKEN' \
'https://api.mercadopago.com/v1/payment_methods' \
```
]]]

> To offer **Pagamento via boleto bancário**, follow the steps below.

Reviewer: ```suggestion
To offer **payments via boleto bancário**, follow the steps below.
```

[[[
```javascript

(async function getIdentificationTypes() {
try {
const identificationTypes = await mp.getIdentificationTypes();
const identificationTypeElement = document.getElementById('form-checkout__identificationType');

createSelectOptions(identificationTypeElement, identificationTypes);
} catch (e) {
return console.error('Error getting identificationTypes: ', e);
}
})();

function createSelectOptions(elem, options, labelsAndKeys = { label: "name", value: "id" }) {
const { label, value } = labelsAndKeys;

elem.options.length = 0;

const tempOptions = document.createDocumentFragment();

options.forEach(option => {
const optValue = option[value];
const optLabel = option[label];

const opt = document.createElement('option');
opt.value = optValue;
opt.textContent = optLabel;

tempOptions.appendChild(opt);
});

elem.appendChild(tempOptions);
}
```
]]]

## Send payment

When finalizing the inclusion of the payment form and obtaining the types of documents, it is necessary to forward the buyer's email, type and document number, the payment method used and the details of the amount to be paid using our Payments API or one of our SDKs.

> To configure **pagamento via boleto bancárioa**, send a **POST** with the following parameters to the endpoint [/v1/payments](/developers/en/reference/payments/_payments/post) and run the request or, if you prefer, use one of our SDKs below.

Reviewer: ```suggestion
To configure **payments via boleto bancárioa**, send a **POST** with the following parameters to the endpoint [/v1/payments](/developers/en/reference/payments/_payments/post) and run the request or, if you prefer, use one of our SDKs below.
```

| value          | BigDecimal | Property to define the daily interest value to be applied when the payment is made after the due date.                                                                                                                                                                                                                                                                                                                                                          |
| type           | String     | Property to define the type of calculation on the value reported in "value".                                                                                                                                                                                                                                                                                                                                                                                    |

## Expiration date

> The default expiration date for **pagamento com boleto bancário** is 3 days. Optionally, it is possible to change this date by sending the date_of_expiration field in the payment creation request, defining a period between 1 and 30 days from the boleto issuance date.

Reviewer: ```suggestion
The default expiration date for **payments with boleto bancário** is 3 days. Optionally, it is possible to change this date by sending the date_of_expiration field in the payment creation request, defining a period between 1 and 30 days from the boleto issuance date.
```
